EXILED/Exiled.Loader/Loader.cs

```diff
@@ -116,13 +115,30 @@ public Loader()
             .Build();
 
         /// <summary>
-        /// Loads all plugins.
+        /// Loads all plugins, both globals and locals.
         /// </summary>
         public static void LoadPlugins()
         {
             File.Delete(Path.Combine(Paths.Plugins, "Exiled.Updater.dll"));
 
-            foreach (string assemblyPath in Directory.GetFiles(Paths.Plugins, "*.dll"))
+            LoadPluginsFromDirectory();
+            LoadPluginsFromDirectory(Server.Port.ToString());
+        }
+
+        /// <summary>
+        /// Load every plugin inside the given directory, if null it's default EXILED one (global).
+        /// </summary>
+        /// <param name="dir">The sub-directory of the plugin - if null the default EXILED one will be used.</param>
+        public static void LoadPluginsFromDirectory(string dir = null)
```

Comment:
this doesn't need to be public since you're not accessing it from outside of the Loader class. In this instance, private should work just fine.

```suggestion
        private static void LoadPluginsFromDirectory(string dir = null)
```

Reply:
I wanted to make it private but I had an oversight xP